### PR TITLE
Fixes #4982 - add better example

### LINKED ISSUE #4982: add an example of configuration.psd1, explain utility
No configuraiton.psd1 example is a little confusing.

The utility of this is that you don't have people evaling powershell to set values - might be good to connect that.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c854b3ef-0ca6-5f36-f71f-9d112d525394
* Version Independent ID: 589d7d4e-5a40-8fec-4409-c98a001b81b8
* Content: [Import-PowerShellDataFile (Microsoft.PowerShell.Utility)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/import-powershelldatafile?view=powershell-5.1#feedback)
* Content Source: [reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @JamesWTruher
* Microsoft Alias: **jimtru**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes #4982 - add better example
Fixes [AB#1626783](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1626783)

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ external help file: Microsoft.PowerShell.Utility-help.xml
...
+ms.date: 10/25/2019
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/import-powershelldatafile?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: Import-PowerShellDataFile
 ---
 # Import-PowerShellDataFile
 
 ## SYNOPSIS
-Imports values from a .PSD1 file without invoking its contents
+Imports values from a .PSD1 file without invoking its contents.
```
Comment (davidsmatlak) [inline reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md]: Wrap .PSD1 with backticks.

File: reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md [hunk 2]
```diff
@@ -29,28 +29,44 @@ Import-PowerShellDataFile [-LiteralPath <String[]>] [<CommonParameters>]
...
+`.PSD1` file. The values could be imported using `Invoke-Expression` on the contents of the file.
+However, `Invoke-Expression` runs any code contained in the file. This could produce unwanted
+results or execute unsafe code. `Import-PowerShellDataFile` imports the data without invoking the
+code.
 
 ## EXAMPLES
 
 ### Example 1: Retrieve values from PSD1
 
+This examples retrieves the key-value pairs stored in the hashtable kept inside the
```
Comment (davidsmatlak) [inline reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md]: Use singular: example

File: reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md]: fixed

File: reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Utility/Import-PowerShellDataFile.md]: fixed

**Pull request merged**